source/ed/ed_reader.cpp

```diff
@@ -886,6 +887,7 @@ void EdReader::fill_vehicle_journeys(nt::Data& data, pqxx::work& work){
             auto frame_it = this->frame_map.find(const_it["frame_id"].as<idx_t>());
             if(frame_it != this->frame_map.end()) {
                 vj->frame = frame_it->second;
+                vj->frame->vehiclejourney_list.push_back(vj);
```

Comment:
I vote for @antoine-de (even if it hurts), as it is the same problem (and roughly the same amount of requests I guess) than navitia::type::PhysicalMode::get.